packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -22,17 +22,22 @@ import {
 import {repeat} from 'lit/directives/repeat.js';
 import {styleMap} from 'lit/directives/style-map.js';
 import {live} from 'lit/directives/live.js';
-import Minisearch from 'minisearch';
 import type {Drawer} from '@material/mwc-drawer';
 import {animate, Options as AnimationOptions} from '@lit-labs/motion';
+import algoliasearch, {
+  SearchClient,
+  SearchIndex,
+} from 'algoliasearch/dist/algoliasearch-lite.esm.browser.js';
```

Comment:
We do our own bundling with rollup, why do we need to reach into the dist directory for the `algoliasearch` package? Hmm.. it looks like their default export from the package might be cjs so maybe that's why. 😞 I don't know the answer to this but is TS unable to handle that with `esModuleInterop` and/or `allowSyntheticDefaultImports`?

Reply:
We need to reach into dist folder because this is the only proper ESM browser build. And our dev build intentionally does no bundling just serve TSC. That means we can't do weird cjs imports without lots of infra which I didn't want to touch

Reply:
That sounds like a good reason to me!